src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitController.kt

```diff
@@ -494,6 +495,9 @@ class VisitController(
     @Schema(name = "toDate", description = "Get visits to date", example = "2023-05-31", required = true)
     @RequestParam
     toDate: LocalDate,
+    @Schema(name = "visitRestriction", description = "Visit Restriction - OPEN / CLOSED / UNKNOWN", example = "OPEN", required = false)
+    @RequestParam
+    visitRestriction: List<VisitRestriction>?,
```

Comment:
visitRestriction should be visitRestrictions

Reply:
updated